D1673/P1673.md

```diff
@@ -6168,21 +6174,30 @@ template<class ... Indices>
   constexpr index_type operator() (Indices... k) const noexcept;
 ```
 
-[11]{.pnum} *Constraints*:
+[12]{.pnum} *Constraints*:
 
-  * [11.1]{.pnum} `sizeof...(Indices)` equals `extents_type::rank()`,
+  * [12.1]{.pnum} `sizeof...(Indices)` equals `extents_type::rank()` is `true`,
 
-  * [11.2]{.pnum} `(is_convertible_v<Indices, index_Type> && ...)` is `true`, and
+  * [12.2]{.pnum} `(is_convertible_v<Indices, index_Type> && ...)` is `true`, and
 
-  * [11.3]{.pnum} `(is_nothrow_constructible_v<index_type, Indices>)` is `true`.
+  * [12.3]{.pnum} `(is_nothrow_constructible_v<index_type, Indices>)` is `true`.
 
-[12]{.pnum} *Preconditions:* `extents_type::`_`index-cast`_`(k)` is
-    a multidimensional index in _`the-extents`_ (**[mdspan.overview]**).
+[13]{.pnum} *Preconditions:* `extents_type::`_`index-cast`_`(k)` is
+    a multidimensional index in _`extents_`_ ([mdspan.overview]).
 
-[13]{.pnum} *Returns:* Let `N` equal `extent(0)`,
+[14]{.pnum} *Returns:* Let `N` equal `extent(0)`,
     let `i` be the zeroth element of `k`,
-    and let `j` be the first element of `k`.  Then:
+    and let `j` be the first element of `k`. Then
+
+  * [14.1]{.pnum} `*this(j, i)` if `i > j` is `true`; otherwise
```

Comment:
Do we need parentheses around `*this`?  (I find it more clear to include them.)

```suggestion
  * [14.1]{.pnum} `(*this)(j, i)` if `i > j` is `true`; otherwise
```